src/common/live-chat-redirection-checking.js

```diff
@@ -0,0 +1,12 @@
+export const checkLiveChatRedirection = () => {
+    if (typeof window !== 'undefined') {
+        const live_chat_key = 'live_chat_redirection'
```

Comment:
Maybe we can declare this variable in `constants.js` and use it also in `utility.js`.

Reply:
fixed in latest commit.